### PR TITLE
[DSD-4835] Updated Images as per requirment

### PR DESCRIPTION
**Pull request by abhishek8shankar**

### CODE REVIEW

File: deployment/v3/mosip/ida/install.sh [hunk 1]
```diff
@@ -26,13 +26,13 @@ function installing_ida() {
   helm -n $NS install ida-keygen mosip/keygen --set image.repository=mosipqa/keys-generator --set image.tag=develop --wait --wait-for-jobs  --version $KEYGEN_CHART_VERSION -f keygen_values.yaml
 
   echo Installing ida auth
-  helm -n $NS install ida-auth mosip/ida-auth --set image.repository=mosipqa/authentication-service --set image.tag=release-1.2.0.1 --version $CHART_VERSION
+  helm -n $NS install ida-auth mosip/ida-auth --set image.repository=mosipid/authentication-service --set image.tag=1.2.0.1-B6 --version $CHART_VERSION
 
   echo Installing ida internal
-  helm -n $NS install ida-internal mosip/ida-internal --set image.repository=mosipqa/authentication-internal-service --set image.tag=release-1.2.0.1 --version $CHART_VERSION
+  helm -n $NS install ida-internal mosip/ida-internal --set image.repository=mosipid/authentication-internal-service --set image.tag=release-1.2.0.1-B6 --version $CHART_VERSION
```
Comment (Prafulrakhade) [inline deployment/v3/mosip/ida/install.sh:32]: 1.2.0.1-B6

File: deployment/v3/mosip/idrepo/install.sh [hunk 2]
```diff
@@ -25,13 +25,13 @@ function installing_idrepo() {
   helm -n $NS install idrepo-saltgen  mosip/idrepo-saltgen --set image.repository=mosipqa/id-repository-salt-generator --set image.tag=develop --version $CHART_VERSION --wait --wait-for-jobs
```
Comment (Prafulrakhade) [inline deployment/v3/mosip/idrepo/install.sh:25]: --set image.repository=mosipid/id-repository-salt-generator --set image.tag=1.2.0.1-B3

File: deployment/v3/mosip/idrepo/install.sh [hunk 3]
```diff
@@ -25,13 +25,13 @@ function installing_idrepo() {
...
 
   echo Running credential
-  helm -n $NS install credential mosip/credential --set image.repository=mosipqa/credential-service --set image.tag=release-1.2.0.1-B3--version $CHART_VERSION
+  helm -n $NS install credential mosip/credential --set image.repository=mosipid/credential-service --set image.tag=release-1.2.0.1-B3--version $CHART_VERSION
```
Comment (Prafulrakhade) [inline deployment/v3/mosip/idrepo/install.sh:28]: remove release only 1.2.0.1-B3 required and also give one space after that

File: deployment/v3/mosip/ida/install.sh (hunk 1 above)
Comment (Prafulrakhade) [reply deployment/v3/mosip/ida/install.sh:32]: also give one space after that

File: deployment/v3/mosip/idrepo/install.sh [hunk 4]
```diff
@@ -25,13 +25,13 @@ function installing_idrepo() {
...
 
   echo Running credential request service
-  helm -n $NS install credentialrequest mosip/credentialrequest --set image.repository=mosipqa/credential-request-generator --set image.tag=release-1.2.0.1-B3 --version $CHART_VERSION
+  helm -n $NS install credentialrequest mosip/credentialrequest --set image.repository=mosipid/credential-request-generator --set image.tag=release-1.2.0.1-B3 --version $CHART_VERSION
```
Comment (Prafulrakhade) [inline deployment/v3/mosip/idrepo/install.sh:31]: remove release

File: deployment/v3/mosip/idrepo/install.sh [hunk 5]
```diff
@@ -25,13 +25,13 @@ function installing_idrepo() {
...
 
   echo Running identity service
-  helm -n $NS install identity mosip/identity --set image.repository=mosipqa/id-repository-identity-service --set image.tag=release-1.2.0.1-B3 --version $CHART_VERSION
+  helm -n $NS install identity mosip/identity --set image.repository=mosipid/id-repository-identity-service --set image.tag=release-1.2.0.1-B3 --version $CHART_VERSION
```
Comment (Prafulrakhade) [inline deployment/v3/mosip/idrepo/install.sh:34]: remove release

File: deployment/v3/mosip/ida/install.sh [hunk 6]
```diff
@@ -26,13 +26,13 @@ function installing_ida() {
...
 
   echo Installing ida otp
-  helm -n $NS install ida-otp mosip/ida-otp --set image.repository=mosipqa/authentication-otp-service --set image.tag=release-1.2.0.1 --version $CHART_VERSION
+  helm -n $NS install ida-otp mosip/ida-otp --set image.repository=mosipid/authentication-otp-service --set image.tag=release-1.2.0.1-B6 --version $CHART_VERSION
```
Comment (Prafulrakhade) [inline deployment/v3/mosip/ida/install.sh:35]: remove release

File: deployment/v3/mosip/ida/install.sh [hunk 7]
```diff
@@ -26,13 +26,13 @@ function installing_ida() {
   helm -n $NS install ida-keygen mosip/keygen --set image.repository=mosipqa/keys-generator --set image.tag=develop --wait --wait-for-jobs  --version $KEYGEN_CHART_VERSION -f keygen_values.yaml
```
Comment (Prafulrakhade) [inline deployment/v3/mosip/ida/install.sh:26]: mosipid/keys-generator:1.2.0.1-B4

File: deployment/v3/mosip/keymanager/install.sh [hunk 8]
```diff
@@ -26,7 +26,7 @@ function installing_keymanager() {
   helm -n $NS install kernel-keygen mosip/keygen --set image.repository=mosipqa/keys-generator --set image.tag=develop --wait --wait-for-jobs --version $CHART_VERSION -f keygen_values.yaml
```
Comment (Prafulrakhade) [inline deployment/v3/mosip/keymanager/install.sh:26]: mosipid/keys-generator:1.2.0.1-B4

File: deployment/v3/mosip/keymanager/install.sh [hunk 9]
```diff
@@ -26,7 +26,7 @@ function installing_keymanager() {
...
 
   echo Installing keymanager
-  helm -n $NS install keymanager mosip/keymanager --set image.repository=mosipqa/kernel-keymanager-service --set image.tag=release-1.2.0.1 --version $CHART_VERSION
+  helm -n $NS install keymanager mosip/keymanager --set image.repository=mosipid/kernel-keymanager-service --set image.tag=release-1.2.0.1-B4 --version $CHART_VERSION
```
Comment (Prafulrakhade) [inline deployment/v3/mosip/keymanager/install.sh:29]: remove release

File: deployment/v3/mosip/ida/install.sh (hunk 7 above)
[the remark this answers is not in the document]
Comment (abhishek8shankar) [reply deployment/v3/mosip/ida/install.sh:26]: updated

**Review by ckm007 — APPROVED** [review]

**Pull request merged**